Source/ChocolateyGui/App.xaml.cs

```diff
@@ -21,7 +22,20 @@ public partial class App
     {
         static App()
         {
-            Container = AutoFacConfiguration.RegisterAutoFac();
+            Container = AutoFacConfiguration.InitialRegistration();
+
+            var configurationProvider = Container.Resolve<IChocolateyConfigurationProvider>();

```

Comment:
So, are you saying just to "new" up an instance on IChocolateyConfigurationProvider, within the AutoFacConfiguration class, and then do the remaining registration?  I need an instance of that class, in order to know what version of Chocolatey is being used.